samples/SampleApp/Startup.cs

```diff
@@ -73,7 +73,19 @@ public static Task Main(string[] args)
                     options.Listen(IPAddress.Loopback, basePort + 1, listenOptions =>
                     {
                         listenOptions.UseHttps("testCert.pfx", "testPassword");
-                        listenOptions.UseConnectionLogging();
+                    });
+
+                    options.ListenLocalhost(basePort + 2);
+
+                    // Use the default certificate
+                    options.ListenLocalhost(basePort + 3, useHttps: true);
+
+                    // Use the default certificate
+                    options.ListenPrefix($"https://*:{basePort + 4}");
```

Comment:
What if you want to use the default certificate with `Listen(IPEndPoint)` or `ListenLocalhost`. Are you forced to convert your endpoint to a string and use `ListenPrefix`.

Why not just add a `UseHttps` overload that uses the default certificate? This way you can remove ListenPrefix entirely, and instead of adding `ListenType.Prefix` you can just add `ListenType.Loopback` which I prefer greatly.